qpp-data-restoration/.gitignore

```diff
@@ -0,0 +1,7 @@
+target/
```

Comment:
Doesn't our top-level `.gitignore` file cover all of these?  It should cascade to the sub-directories.